client/components/Main/ListGroupItem.tsx

```diff
@@ -265,3 +274,5 @@ export class ListGroupItem extends React.Component<IProps, IState> {
         return null;
     }
 }
+
+export const ListGroupItem = connect(mapStateToProps)(ListGroupComponent);
```

Comment:
It's not a good idea to connect list items to the Redux store. If the list gets big, then  there will be 1 store listener per list item.

It would be better for performance to add the selector/param to a higher up component